### PR TITLE
Hifi About page

### PR DESCRIPTION
**Pull request by ItsAdel**
## A brief summary of what changes were introduced.

Added hifi about page

## Any additional things to take into consideration.

Did not add the light glowing onto the about panel

## :flashlight: Testing Instructions

See changes in aplify

### CODE REVIEW

**Review by rmehri01 — APPROVED** [review]
Looks great!

**Comment by kozr** [comment]
image
Did the hero section miss the bottom of the bubbly clouds? Figma is above and amplify is below
image

File: components/About.vue [hunk 1]
```diff
@@ -1,63 +1,141 @@
...
-          Your health and well being is our top priority,  nwHacks and nwPlus will never endorse activities that might put you or others in danger.
-        </p>
-      </div>
-      <div class="about-box about-box3">
-        <p class="about-p about-p3">
-          If you have any thoughts on the matter or wish to express your opinion please don't hesitate to send us an email to <span class="about-p about-p-bold">logistics@nwplus.io</span>
-        </p>
+      <span class="about-text">About</span>
+      <p class="about-p1">
+        Last year we connected over 800 of the brightest developers, engineers, and designers in the Pacific Northwest collaborate and create amazing projects. This year we want to take it to the next level and connect even more hackers across the globe from the comfort of your home. **<span class="about-p-bold">nwHacks is going online</span>**!
```
Comment (kozr) [inline components/About.vue]: Can we remove the **?

**Comment by kozr** [comment]
The css must've been a lot of work. Thanks, Adel. It looks wonderful!

File: components/About.vue (hunk 1 above)
Comment (ItsAdel) [reply components/About.vue]: yup, done

**Comment by ItsAdel** [comment]
> Did the hero section miss the bottom of the bubbly clouds? Figma is above and amplify is below

@kozr  No, the image includes the bottom. It is getting cut off at the bottom of the hero section so we don't see the full background. It's hard to create that cloudy transition between two different components. We might have to experiment with making a separate svg for the cloudy bit and try to insert it between the two components.

**Pull request merged**